components/Card/CardTitle.tsx

```diff
@@ -27,93 +13,151 @@ interface CardTitleProps {
   inHeaderElement?: ReactElement
   isBillMatch?: boolean
   isUserMatch?: boolean
+  type?: string
 }
 
 export const CardTitle = (props: CardTitleProps) => {
```

Comment:
Including `isBillMatch/isUserMatch` makes this very specific to Newsfeed components for a component as generic as `Card/CardTitle` - could we use a more specific name here (or maybe build the `NewsfeedCard` components on top of generic `Card` components with fewer props)?